attest/enclave-api/src/error.rs

```diff
@@ -127,3 +134,15 @@ impl From<ParseSealedError> for Error {
         Error::Unseal(src)
     }
 }
+
+impl From<mc_util_serial::encode::Error> for Error {
+    fn from(src: mc_util_serial::encode::Error) -> Self {
+        Error::Encode(format!("{}", src))
```

Comment:
:upside_down_face: Could use `to_string()`, requires the `ToString` trait, `use alloc::string::ToString;`
not sure if it's any better, seeing a single brace in the format feels like a code smell
```suggestion
        Error::Encode(src.to_string())
```

Reply:
Done.